docs/src/content/docs/environmental-impact.md

```diff
@@ -89,20 +89,21 @@ Choosing a host that uses renewable energy will mean lower carbon emissions for
 Curious how other docs frameworks compare?
 These tests with the [Website Carbon Calculator][wcc] compare similar pages built with different tools.
 
-| Framework                   | CO₂ per page visit |
-| --------------------------- | ------------------ |
-| [Starlight][sl-carbon]      | 0.01g              |
-| [VitePress][vp-carbon]      | 0.05g              |
-| [Docus][dc-carbon]          | 0.05g              |
-| [Sphinx][sx-carbon]         | 0.07g              |
-| [MkDocs][mk-carbon]         | 0.10g              |
-| [Nextra][nx-carbon]         | 0.11g              |
-| [docsify][dy-carbon]        | 0.11g              |
-| [Docusaurus][ds-carbon]     | 0.24g              |
-| [Read the Docs][rtd-carbon] | 0.24g              |
-| [GitBook][gb-carbon]        | 0.71g              |
-
-<small>Data collected on 14 May 2023. Click a link to see up-to-date figures.</small>
+| Framework                   | CO₂ per page visit | Rating |
+| --------------------------- | ------------------ | :----: |
+| [Starlight][sl-carbon]      | 0.01g              |   A+   |
+| [Read the Docs][rtd-carbon] | 0.05g              |   A+   |
+| [Sphinx][sx-carbon]         | 0.06g              |   A+   |
+| [VitePress][vp-carbon]      | 0.07g              |   A+   |
+| [Docus][dc-carbon]          | 0.09g              |   A+   |
+| [docsify][dy-carbon]        | 0.10g              |   A    |
+| [Nextra][nx-carbon]         | 0.11g              |   A    |
+| [MkDocs][mk-carbon]         | 0.19g              |   B    |
+| [Docusaurus][ds-carbon]     | 0.21g              |   B    |
+| [GitBook][gb-carbon]        | 0.43g              |   C    |
+| [Mintlify][mt-carbon]       | 1.22g              |   F    |
+
+<small>Data collected on 22 July 2024. Click a link to see up-to-date figures.</small>
```

Comment:
```suggestion
<small>Data collected on 22 July 2024. Click a respective framework's link to see up-to-date figures.</small>
```
 Makes it a little more clear which link is being referred to. But may already be clear enough in this context, up to you!

Reply:
Yeah, I suspect it's clear enough already. Although now you have me second guessing whether "click" is the right action at all if someone is on a touch device etc. 😅

Going to merge without this so I don't need to get into that, but thanks nonetheless!